SB2018/2018-03-09.md

```diff
@@ -8,7 +8,7 @@ Hi!
 We're happy to announce a new update to our kits!
 
 This release contains many improvements to vision (other changes too):
-- Completely rewrite vision to greatly increase its accuracy (now all Cartesian values have an error of only 15% of their total distance).
+- Completely rewrite vision to greatly increase its accuracy (now all Cartesian values have an tolerance of only 15% of their total distance).
```

Comment:
"an" tolerance?

In fact, thinking about this in the wider context of the email, do we even need this level of detail in the email? (I'm not sure how detailed you're aiming for) Would it make sense to just say:
> Completely rewrite vision to greatly increase its accuracy